src/ToDo.UI/Strings/fr/Resources.resw

```diff
@@ -1,5 +1,8 @@
 <root>
```

Comment:
Just need to add the standard resources.resw header so that the file can be opened in VS 😉